### PR TITLE
Main banner

### LINKED ISSUE #99: MainPage: Index Banner for main view
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
Index banner component for the main view
image

Button:
Buttons labels and action url should be stored in json
Primary button: Get on Board -> scroll to cards section below (anchor #onboard)
Secondary button: Watch Video

Cards:
We will need 4 cards total.
Make sure content will be grabbed from JSON: image_url, title, description, button label
Content for new cards can be found here: https://docs.google.com/document/d/1spHRz5QpyGWRx00ljAMkF2Ek6AnoQC7Ow-WAaOfBvvQ/edit#heading=h.63dy83a99wh

requires #97 based on #83

### PR DESCRIPTION
**Pull request by PabloAlice**
fix #99

### CODE REVIEW

**Comment by menschikov** [comment]
@skkr please check styles here 
image

**Comment by menschikov** [comment]
@PabloAlice cards are wrong too

**Comment by menschikov** [comment]
@PabloAlice please make this cards look good
image

I will use 2 line header and 2 line text everywhere to adjust them vertically. But please apply next styles:
1) button width: 100%
2) card content (h,p, button) text-align: center
3) margin to for <p> 1rem;

@skkr maybe now cards fit framework styles but they not looks good together. We need to apply these small changes.

@PabloAlice @MatiasOS , this was mentioned already: LearnMoreCards and Banner cards should get content from same place. It's basically one card with two appearances: short (in banner) and full (in footer). Please make them get content from same file, this is merged structure:

 {
    title: 'For Suppliers',
    text: 'Suppliers can distribute inventory directly to points of sale, skipping costly intermediaries and bottlenecks to data.',
    **shortText: 'New profitable distribution channel'**
    linkText: 'Learn More',
    **buttonLabel: 'Learn More"**
    href: '/suppliers',
    imgSrc: suppliersImg,
  },

**Comment by skkr** [comment]
@menschikov We have different card styles for different situations, for this we should keep the original card style as in the footer.

If we need to change the look of something, it must be a design requirement prior to the implementation instance, so is clear for devs what assets to use from wt-ui.

@PabloAlice please use the original style. With a link instead a button, if there is a reason to change the design we can evaluate later, with more time, what design solution fits best.

**Comment by skkr** [comment]
That's true, but UX requests should be related of the content that needs to be shown, and UI to design and look and feel. 

Based on that, my advice is to use the original cleaner style on cards.

Alternatively we have a card style with buttons. If that style is used, is ok to change the button tyoe to outline when necessary.

But again, design must come before implementation. Doing design changes on the implementation and adding them to the framework later is not an approach we should follow.

**Comment by skkr** [comment]
@PabloAlice Alex desicion is to make the changes from [his comment avobe] (https://github.com/windingtree/homepage/pull/152#issuecomment-450466501) please do that.

**Comment by menschikov** [comment]
@skkr can you please adjust image position and menu transparency?

**Comment by skkr** [comment]
@menschikov fyi - general styles for the hero are fixed, I want to review some things before pushing but I think is ok. Please check the message I sent you about card styles. Depending on the style we use, @MatiasOS might need to do some adjustments to the component.

I also created some styles for the header transition, but it will require to modify the navbar wich is a component from wt-ui-react. Mati, we can discuss it in a quick call.

File: src/features/Main/index.jsx [hunk 1]
```diff
@@ -11,45 +11,43 @@ type PropsType = {
 };
```
Comment (skkr) [inline src/features/Main/index.jsx]: @PabloAlice we must keep the structure as was defined in wt-ui. I'll revert the markup and will put comments so you or @MatiasOS can update the script for the transition.

**Comment by skkr** [comment]
Navbar animation is enabled.

**Comment by menschikov** [comment]
there's one thing with cards in top and bottom banner (For Sellers, For suppliers etc). There should be as many cards as JSON objects. Now we have 5 JSON objects but top banned shows just 4. Actually I need 4 cards for now, but this is bug in general, the number of cards should be same in both banners

**Pull request merged**